youki_integration_test/src/support.rs

```diff
@@ -43,29 +51,6 @@ pub fn generate_uuid() -> Uuid {
     }
 }
 
-pub fn test_builder(status: bool, name: &str, diagnostic: &str) -> TapTest {
-    TapTestBuilder::new()
-        .name(name)
-        .passed(status)
-        .diagnostics(&[diagnostic])
-        .finalize()
-}
-
-pub fn print_test_results(test_name: &str, tests: Vec<TapTest>) {
-    let tap_suite = TapSuiteBuilder::new()
-        .name(test_name)
-        .tests(tests)
-        .finalize();
-
-    // print to stdout
-    println!("# Start {}", test_name);
-    match tap_suite.print(io::stdout().lock()) {
-        Ok(_) => {}
-        Err(reason) => eprintln!("{}", reason),
-    }
-    println!("\n# End {}", test_name);
-}
-
 // Temporary files to be used for testing are created in the `integration-workspace`.
 fn prepare_test_workspace(project_path: &Path) -> Result<(), std::io::Error> {
     let integration_test_workspace_path = project_path.join("integration-workspace");
```

Comment:
It's better to include UUID, timestamp, etc. to avoid conflicts with other tests.

Reply:
We can add that, but currently we are using this function once before all the test begins, so it is common for all tests, I don't think there will be conflicts. For the change we'll need something similar to get/set runtime path, as currently the integration workspace name is hardcoded everywhere.

Reply:
I wondered if this would be a problem if I had to run the tests in parallel. Well, we can fix it later.